Day5_excercise_2.rb

```diff
@@ -0,0 +1,79 @@
+#1. Create an array to store 5 names. Have the user enter in the 5 names and then print out the 5 names in all UPPERCASE all on the same line.
+# Note that the user may not enter all the names in uppercase.
+names = Array.new
+5.times do
+  puts "Enter your name: "
+  names << gets.chomp
+end
+print names.map { |names| names.upcase }
+
+#2. Create an array to store 3 boolean values entered in by the user.
+#Print out YES if all the values are true and NO if at least one value is false.
+#Hint: Prompt the user to enter true or false, and accept those values using gets.
+#Since gets will give us back a string (instead of the boolean values we want) use
+#if statements to convert those strings to boolean values.
+
+
+#3. Create an array to store the following words: firetruck, fire drill, fire hydrant, firefighter, fireproof, fire station, fire hose.
+#Then write code that uses the array to print out truck, drill, hydrant, fighter, proof, station, hose without modifying the array.
+list = ["firetruck", "fire drill", "fire hydrant", "firefighter", "fireproof", "fire station", "fire hose"]
+
+print list[0][4..8] + list[1][4..9] + list[2][4..11] + " " + list[3][4..10] + " " + list[4][4..8] + " " + list[5][4..11] + list[6][4..9]
```

Comment:
You missed the comma from printed message

Reply:
This is the printed result `truck drill hydrant fighter proof  station hose`, doesn't look right.